src/script.c

```diff
@@ -1235,6 +1235,32 @@ int wally_witness_p2wpkh_from_sig(
     return ret;
 }
 
+int wally_witness_p2tr_from_sig(
+    const unsigned char *sig,
+    size_t sig_len,
+    struct wally_tx_witness_stack **witness)
+{
```

Comment:
nit: prefer:
```
int wally_witness_p2tr_from_sig(const unsigned char *sig, size_t sig_len,
                                struct wally_tx_witness_stack **witness)
```
The one-line per parameter format is for the headers, in the impls I like to preseve vertical space for easier reviewing if possible.